core/server/common/src/main/java/alluxio/master/journal/noop/NoopJournalSystem.java

```diff
@@ -57,4 +57,9 @@ public void start() {}
 
   @Override
   public void stop() {}
+
+  @Override
+  public void checkpoint() {
+    throw new UnsupportedOperationException("Checkpoint is not supported in NoopJournalSystem");
```

Comment:
more consistent to make this a no-op like other methods

Reply:
Changed, thanks